lib/ibm_speech_to_text.ex

```diff
@@ -68,20 +67,18 @@ defmodule Membrane.IBMSpeechToText do
   end
 
   @impl true
-  def handle_playing(_ctx, state) do
-    with {:ok, pid} <- Client.start_link(state.region, state.api_key, state.client_options) do
+  def handle_playing(ctx, state) do
+    client = %{
+      id: Client,
+      start: {Client, :start_link, [state.region, state.api_key, state.client_options]}
+    }
+
+    with {:ok, pid} <- UtilitySupervisor.start_link_child(ctx.utility_supervisor, client) do
```

Comment:
I am think that in case start_link_child fails, we should explicitly raise but is this even possible?